web/src/app/util/RequireConfig.js

```diff
@@ -60,7 +60,9 @@ function parseValue(type, value) {
 
 function isTrue(value) {
   if (Array.isArray(value)) return value.length > 0
-
+  if (value === 'true' || value === 'false') {
+    return value === 'true'
+  }
```

Comment:
This is just a nit, but the only exception we really need to code for is `'false'`

`if (value === 'false') return false`